src/c/src/zookeeper.c

```diff
@@ -4358,7 +4358,9 @@ static const char* format_endpoint_info(const struct sockaddr_storage* ep)
 {
     static char buf[128] = { 0 };
     char addrstr[128] = { 0 };
+    const char *fmtstring;
     void *inaddr;
+    int inet6 = 0;  // poor man's boolean
```

Comment:
Just a nitpick for future readability - Maybe just java speaking from me, but why not use some more clear name like is_inet6? 
(Or include stdbool)

Reply:
Sounds good to me, I'll rework that variable.